templates/puppet.conf.erb

```diff
@@ -64,3 +64,6 @@
 <% scope.lookupvar("puppet::additional_settings").sort_by {|k, v| k}.each do |key, value| -%>
     <%= key %> = <%= value %>
 <% end -%>
+<% if @stringify_facts == false -%>
```

Comment:
I think this is OK, as the default is ```true``` now and the whole option will be gone in Puppet 4 by puppetlabs/puppet#3212